### PR TITLE
hcloud 1.8.0

### PR DESCRIPTION
**Pull request by LKaemmerling**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
https://github.com/hetznercloud/cli/releases/tag/v1.8.0

Since we now uses golang 1.11 we does not need to overwrite the GOPATH Env anymore.

### CODE REVIEW

File: Formula/hcloud.rb [hunk 1]
```diff
@@ -15,7 +15,6 @@ class Hcloud < Formula
   depends_on "go" => :build
 
   def install
-    ENV["GOPATH"] = buildpath
```
Comment (commitay) [inline Formula/hcloud.rb]: `ENV["GOPATH"] = HOMEBREW_CACHE/"go_cache"`

Comment (LKaemmerling) [reply Formula/hcloud.rb]: Changed.

**Comment by fxcoudert** [comment]
Thanks @LKaemmerling

**Comment by fxcoudert** [comment]
Bottles could not be uploaded:

```
+ brew test-bot --ci-upload
cp /var/lib/jenkins/jobs/Homebrew Core Pull Requests/configurations/axis-version/sierra/builds/30583/archive/hcloud--1.8.0.sierra.bottle.json /var/lib/jenkins/jobs/Homebrew Core Pull Requests/configurations/axis-version/sierra/builds/30583/archive/hcloud--1.8.0.sierra.bottle.tar.gz /var/lib/jenkins/jobs/Homebrew Core Pull Requests/configurations/axis-version/mojave_testing/builds/30583/archive/hcloud--1.8.0.mojave.bottle.json /var/lib/jenkins/jobs/Homebrew Core Pull Requests/configurations/axis-version/mojave_testing/builds/30583/archive/hcloud--1.8.0.mojave.bottle.tar.gz /var/lib/jenkins/jobs/Homebrew Core Pull Requests/configurations/axis-version/high_sierra/builds/30583/archive/hcloud--1.8.0.high_sierra.bottle.json /var/lib/jenkins/jobs/Homebrew Core Pull Requests/configurations/axis-version/high_sierra/builds/30583/archive/hcloud--1.8.0.high_sierra.bottle.tar.gz /var/lib/jenkins/jobs/Homebrew Core Pull Requests/configurations/axis-version/el_capitan/builds/30583/archive/hcloud--1.8.0.el_capitan.bottle.tar.gz /var/lib/jenkins/jobs/Homebrew Core Pull Requests/configurations/axis-version/el_capitan/builds/30583/archive/hcloud--1.8.0.el_capitan.bottle.json /var/lib/jenkins/workspace/Homebrew Bottles
Error: undefined method `deep_merge_hashes' for Homebrew:Module
/var/lib/jenkins/workspace/Homebrew Bottles/Library/Taps/homebrew/homebrew-test-bot/cmd/brew-test-bot.rb:1351:in `block in test_ci_upload'
/var/lib/jenkins/workspace/Homebrew Bottles/Library/Taps/homebrew/homebrew-test-bot/cmd/brew-test-bot.rb:1350:in `each'
/var/lib/jenkins/workspace/Homebrew Bottles/Library/Taps/homebrew/homebrew-test-bot/cmd/brew-test-bot.rb:1350:in `reduce'
/var/lib/jenkins/workspace/Homebrew Bottles/Library/Taps/homebrew/homebrew-test-bot/cmd/brew-test-bot.rb:1350:in `test_ci_upload'
/var/lib/jenkins/workspace/Homebrew Bottles/Library/Taps/homebrew/homebrew-test-bot/cmd/brew-test-bot.rb:1644:in `test_bot'
/var/lib/jenkins/workspace/Homebrew Bottles/Library/Taps/homebrew/homebrew-test-bot/cmd/brew-test-bot.rb:1752:in `<top (required)>'
/var/lib/jenkins/workspace/Homebrew Bottles/Library/Homebrew/vendor/portable-ruby/2.3.7/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/var/lib/jenkins/workspace/Homebrew Bottles/Library/Homebrew/vendor/portable-ruby/2.3.7/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
/var/lib/jenkins/workspace/Homebrew Bottles/Library/Homebrew/utils.rb:19:in `require?'
/var/lib/jenkins/workspace/Homebrew Bottles/Library/Homebrew/brew.rb:95:in `<main>'
```

**Comment by LKaemmerling** [comment]
@fxcoudert so what can i do here?

**Comment by fxcoudert** [comment]
We had a little trouble with our CI, but I will re-run and merge soon :)

**Pull request closed without merging**